recipes/autoconf-archive/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+
+
+class AutoconfArchiveConan(ConanFile):
+    name = "autoconf-archive"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/autoconf-archive/"
+    license = "GPL-2.0-or-later"
+    description = "The GNU Autoconf Archive is a collection of more than 500 macros for GNU Autoconf"
+    topics = ("conan", "GNU", "autoconf", "autoconf-archive", "macro")
+    settings = "os"
+    _source_subfolder = "source_subfolder"
+    _autotools = None
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("only Linux is supported")
```

Comment:
Why are you discarding other operating systems? Not tested or known to not work?
Our autoconf/automake/libtool recipes work, so I would be surprised if this package won't.
(Agreed, some macros won't)

Reply:
I'm not really well-versed in other operating systems, especially Windows. So I'll definitely need help with that.